### ISSUE TITLE
Term merge: UBC13-MMS2 complex (GO:0031372) & UBC13-UEV1A complex (GO:0035370)

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-ontology#27697]
I believe these two complexes are referring to the same thing:

id: GO:0031372
name: UBC13-MMS2 complex
def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains, which act as a signal to promote error-free DNA postreplication repair; in Saccharomyces the complex comprises Ubc13p and Mms2p." [GOC:mah, PMID:15772086]
is_a: GO:0031371 ! ubiquitin conjugating enzyme complex

id: GO:0035370
name: UBC13-UEV1A complex
def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains and is involved in NF-kappaB activation. In humans at least, the complex comprises the ubiquitin-conjugating enzyme UBC13 and ubiquitin-conjugating enzyme variant 1A (UEV1A)." [GOC:amm, PMID:16129784]
is_a: GO:0031371 ! ubiquitin conjugating enzyme complex
created_by: bf
creation_date: 2010-03-11T10:53:09Z

Note that:
- yeast Ubc13p = UBE2N/UBE2NL
- yeast Mms2p = UBE2V1/UBE2V2
- UBC13 is an alias for human UBE2N
- UEV1A is an alias for human UBE2V1
- MMS2 (see below) is an alias for UBE2V2

And the abstract of PMID:16129784 (used for the def of 'UBC13-UEV1A complex') says:
Ubc13, a ubiquitin-conjugating enzyme (Ubc), requires the presence of a Ubc variant (Uev) for polyubiquitination. Uevs, although resembling Ubc in sequence and structure, lack the active site cysteine residue and are catalytically inactive. The yeast Uev (Mms2) incites noncanonical Lys63-linked polyubiquitination by Ubc13, whereas the increased diversity of Uevs in higher eukaryotes suggests an unexpected complication in ubiquitination. In this study, we demonstrate that divergent activities of mammalian Ubc13 rely on its pairing with either of two Uevs, Uev1A or Mms2. Structurally, we demonstrate that Mms2 and Uev1A differentially modulate the length of Ubc13-mediated Lys63-linked polyubiquitin chains. Functionally, we describe that Ubc13-Mms2 is required for DNA damage repair but not nuclear factor kappaB (NF-kappaB) activation, whereas Ubc13-Uev1A is involved in NF-kappaB activation but not DNA repair.

Merged def could be:
def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains. In Saccharomyces cerevisiae, the complex comprises Ubc13p and Mms2p, whereas in human it comprises UBE2N and UBE2V1/UBE2V2." [GOC:sm, PMID:15772086, PMID:15772086]

**Comment by sjm41** [comment]
EXP annotations:

**UBC13-MMS2 complex (GO:0031372):**
human: UBE2N, UBE2V2
Arabidopsis: UBC35, UEV1A, UEV1B, UEV1C

**UBC13-UEV1A complex (GO:0035370):**
human: UBE2N, UBE2V1
C elegans: ubc-13, uev-1
fly: ben (= UBC13)

**Comment by pgaudet** [comment]
@sylvainpoux @tberardini @vanaukenk 

Do you agree with this merge? 

Thanks, Pascale

**Comment by sylvainpoux** [comment]
Hi @pgaudet 
I see that ComplexPortal made 2 complexes. And if you look to the UBE2N entry in Swiss-Prot (https://www.uniprot.org/uniprotkb/P61088/entry), you can see that these are definitively 2 different complexes.

[CPX-485](https://www.ebi.ac.uk/complexportal/complex/CPX-485)﻿, UBC13-UEV1A ubiquitin-conjugating enzyme E2 complex
[CPX-530](https://www.ebi.ac.uk/complexportal/complex/CPX-530)﻿, UBC13-MMS2 ubiquitin-conjugating enzyme E2 complex

Then, it depends what you want to capture in GO and with the definition you will give to the parent complex

Sylvain

**Comment by sjm41** [comment]
Right, there are two distinct complexes in CP for humans since there are two MMS2/UEV1A orthologs - now called UBE2V1 & UBE2V2.
But yeast, worms and flies only have one of these genes (while Arabidopsis seems to have 4 copies), so it's unclear whether complexes from those species should be annotated to 'UBC13-MMS2 complex' or 'UBC13-UEV1A complex'.

For GO, we either need a single species-agnostic term here for what appears to be essentially the same complex with the same MF, or we'll need to keep the two current terms, make a new generic parent, and possibly make new terms to accommodate the additional Arabidopsis variants.

I think the former option is preferable?

====

MMS2/UBE2V:
- cerevisiae: Mms2
- Arabidopsis: MMZ1/MMZ2/MMZ3/UEV1D-4
- worm: uev-1
- fly: Uev1A
- human: UBE2V1/UBE2V2

**Comment by tberardini** [comment]
I like "a single species-agnostic term here for what appears to be essentially the same complex with the same MF."  This gets away from encoding the exact names of the components into the term name, given there is variation in the copy number and naming among the different species.

**Comment by sjm41** [comment]
I was just reading the guidelines on GO complexes here: [(https://wiki.geneontology.org/Protein_complexes#Specific_complexes)](https://wiki.geneontology.org/Protein_complexes#Specific_complexes_(%22instances%22))

It says:

**Specific complexes ("instances")**
GO describes general classes of concepts, not specific ones. To describe specific complexes, described by their exact subunits in a specific organisms, can be submitted to [[Complex Portal](https://www.ebi.ac.uk/about/contact/support/complexportal)] and/or [[Protein Ontology (PRO)](https://github.com/PROconsortium/PRoteinOntology/issues/new?assignees=nataled&labels=Term+Request&projects=&template=1--term-request.md&title=Term+issue%3A+). These resources capture complexes with their exact subunit composition (similar to GO annotations).

**Comment by pgaudet** [comment]
OK perfect. Merge it is :)

**Comment by tberardini** [comment]
What will you use as the primary term name?

**Comment by sjm41** [comment]
I'd suggest we keep 'UBC13-MMS2 complex' as the merged term name (since they, at least, are still the current symbols of the yeast proteins) and then include the names of the human gene products in the new def. (I don't think it's feasible the list the symbols in all species in the def.). That is:

name: UBC13-MMS2 complex
def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains. In Saccharomyces cerevisiae, the complex comprises Ubc13p and Mms2p, whereas in human it comprises UBE2N and UBE2V1/UBE2V2." [GOC:sjm, PMID:15772086, PMID:15772086]
is_a: GO:0031371 ! ubiquitin conjugating enzyme complex

**Comment by tberardini** [comment]
Can we have either a nod to such complexes existing in plants/flies or add narrow synonyms that include the names of the proteins/protein complexes in plants and flies? Naming just the human and yeast proteins seems not quite right and would discourage me from using the term, if I wasn't deeply familiar with this subject area.

**Comment by pgaudet** [comment]
Definitely, if these are provided we should also add them if it's useful.

**Comment by sjm41** [comment]
The fly complex has been referred to as 'BEN-UEV1A' (PMID:19141674), so I guess that could be added as a synonym?

**Comment by pgaudet** [comment]
Dear all, 

The proposal has been made to obsolete GO:0035370 UBC13-UEV1A complex. The reason for obsoletion is that this represents the same complex as GO:0031372 UBC13-MMS2 complex, but in some organism, because of additional paralogs, there are multiple forms of the complex. GO:0035370  will be replaced by GO:0031372. 

There are 7 annotations to this term, see https://github.com/geneontology/go-annotation/issues/5202
2 ComplexPortal
2 FlyBase
2 UniProt
1 ZFIN. 

There are no mappings to this term; this term is not in any subset. You can comment on the ticket: https://github.com/geneontology/go-ontology/issues/27697


Thanks, Pascale

**Comment by pgaudet** [comment]
@sjm41 
For the proposed definition cross reference, you have this: ` [GOC:sm, PMID:15772086, PMID:15772086]``
1. As we discussed yesterday, sm is someone else, so I have added 'sjm' as in the users.yaml file.
2. Did you mean to add a different PMID? Now you have the same one twice.

**Comment by pgaudet** [comment]
Changes made to GO:0031372
 name: UBC13-MMS2 complex

+def: "A heterodimeric ubiquitin conjugating enzyme complex that catalyzes assembly of K63-linked polyubiquitin chains. In Saccharomyces cerevisiae, the complex comprises Ubc13p and Mms2p; in human it comprises UBE2N and UBE2V1/UBE2V2; and in plants UBC35/UBC36 and UEV1A/UEV1B/UEV1C/UEV1D-4." [GOC:sjm, PMID:15772086]
+synonym: "BEN-UEV1A complex" EXACT []
+synonym: "ubc-13-uev-1 complex" NARROW []
+synonym: "UBE2N/UBE2NL complex" NARROW []
+synonym: "UBE2V1/UBE2V2 complex" NARROW []
+synonym: "Uev1A-ben complex" NARROW []

Please reopen if this is not right or incomplete.

**Comment by pgaudet** [comment]
I think all is done here.

**Comment by sandraorchard** [comment]
Complex Portal done
